apps/web/utils/debounce.ts

```diff
@@ -0,0 +1,29 @@
+// We need any to avoid code duplication --- we have use cases for
+// both `number` and `strings` in our codebase.
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const debounce = <T extends (...args: any[]) => void>(
```

Comment:
```suggestion
// We need any to avoid code duplication --- we have use cases for
// both `number` and `strings` in our codebase.
// eslint-disable-next-line @typescript-eslint/no-explicit-any
export const debounce = <T extends (...args: any[]) => void>(
```
You can do either `Array<string|number>` or `string[] | number[]` I think? The first one would allow for mixed cases, not sure we need that.

Reply:
It's not working, I already tried. it will crash inside the components